spvc/test/run_spirv_cross_tests.py

```diff
@@ -115,7 +112,7 @@ def remove_files(*filenames):
 # Optionally pass through spirv-opt.
 def compile_input_shader(shader, filename, optimize):
```

Comment:
Pleaes add a method comment to describe the return values.

Reply:
Done